admin_manual/installation/apps_management_installation.rst

```diff
@@ -73,11 +73,12 @@ The configuration example below shows how to add a second directory, called ``ap
 After you add a new directory configuration, you can then move apps from the original app directory to the new one. 
 To do so, follow these steps:
 
-#. Enable maintenance mode.
-#. Disable the apps that you want to move.
+#. :ref:`Enable maintenance mode <maintenance_commands_label>`.
+#. :ref:`Disable the apps <apps_commands_label>` that you want to move.
 #. Create a new apps directory and assign it the same user and group, and ownership permissions as the core apps directory.
 #. Move the apps from the old apps directory to the new apps directory.
 #. Add a new app directory in ``config/config.php``.
+#. If you're using a cache, such as Redis, clear the cache.
 #. Re-enable the apps.
 #. Disable maintenance mode.
 
```

Comment:
I would add that what @VicDeo wrote which helps a lot:

```
The Redis cache could be flushed from CLI with redis-cli tool:

sudo redis-cli
SELECT <dbIndex>
FLUSHDB

Where <dbIndex> - number of Redis database where cache is stored, zero by default at ownCloud

For more information see:
https://redis.io/commands/select
https://redis.io/commands/flushdb
```

Reply:
It's better if that information was included in the relevant section of the documentation and linked to. More than happy to do that.

Reply:
I've documented the basics of clearing Redis and Memcached caches in 17045ef.

Reply:
Great, please add a reference note like "Clearing the Redis Cache" to "If you're using a cache, such as Redis, clear the cache."

Reply:
Meant "Clearing the Cache" to be more open 😄

Reply:
Done. It's covered in #3688.

Reply:
I have seen that it is now described in #3688 (:smile:), but you need to know that this description is available...
My proposal is to have a link in line 81 to this description.

Reply:
You're right. I forgot to do that. Will do that now.

Reply:
👍